### PR TITLE
1.8 release notes: finalize auth items

### PR DESCRIPTION
**Pull request by ericchiang**
cc @kubernetes/sig-auth-pr-reviews 
cc @radhikapc

### CODE REVIEW

File: release-1.8/release_notes_draft.md [hunk 1]
```diff
@@ -431,12 +453,32 @@ kind.
...
 * [beta] Advanced auditing has been promoted from alpha to beta. The webhook and logging policy formats have changed since alpha, and may require modification.
 * [beta] Kubelet certificate rotation through the certificates API has been promoted from alpha to beta. RBAC cluster roles for the certificates controller have been added for common uses of the certificates API, such as the kubelet's.
 * [beta] SelfSubjectRulesReview, an API that lets a user see what actions they can perform with a namespace, has been added to the authorization.k8s.io API group. This bulk query is intended to enable UIs to show/hide actions based on the end user, and for users to quickly reason about their own permissions.
 * [alpha] Building on the 1.7 work to allow encryption of resources such as secrets, a mechanism to store resource encryption keys in external Key Management Systems (KMS) was introduced. This complements the original file-based storage and allows integration with multiple KMS. A Google Cloud KMS plugin was added and will be usable once the Google side of the integration is complete.
 
+#### Other features and bug fixes
```
Comment (luxas) [inline release-1.8/release_notes_draft.md]: we don't have this for any other SIG (so far at least!?)

File: release-1.8/release_notes_draft.md [hunk 2]
```diff
@@ -226,6 +247,7 @@ across the system. Here's the release [scalability validation report].
 ### Auth
 
 - With the introduction of RBAC v1, the RBAC v1alpha1 API group has been deprecated.
+- The API server flag `--experimental-bootstrap-token-auth` is now depricated in favor of `--enable-bootstrap-token-auth`. The `--experimental-bootstrap-token-auth` flag will be removed in 1.9.
```
Comment (luxas) [inline release-1.8/release_notes_draft.md]: nit: deprecated

Comment (liggitt) [inline release-1.8/release_notes_draft.md]: deprecated

File: release-1.8/release_notes_draft.md (hunk 1 above)
Comment (ericchiang) [reply release-1.8/release_notes_draft.md]: All the SIGs are using different formats at the moment.

**Comment by ericchiang** [comment]
Removed the "features" and "other features and bug fixes" sub-headers.

File: release-1.8/release_notes_draft.md (hunk 1 above)
Comment (luxas) [reply release-1.8/release_notes_draft.md]: Indeed :(
Hopefully that will be addressed by the relnotes owners later

**Pull request merged**